src/components/modal/examples/Modal.vue

```diff
@@ -92,7 +94,7 @@ export default {
 },
   data() {
     return {
-      opened: this.$route.name === 'Modals',
+      opened: false,
```

Comment:
I hope this is ok. This works much better for testing state before, during, and after modal is launched.

Reply:
I believe this was in place for backstop, I don't think it was set to open the modal so it was opened on load.

Reply:
Regardless, it's fine since we've stopped using backstop